Makefile

```diff
@@ -48,7 +48,7 @@ export OPERATOR_NAMESPACE?=openshift-compliance
 
 # Operator-sdk variables
 # ======================
-SDK_VERSION?=v0.18.1
+SDK_VERSION?=v0.18.2
```

Comment:
This is OK with me as long as we stay on the 0.18 branch for now. It might be better to skip upgrading for a while unless there is a reason to upgrade and then go directly to 1.0 and figure out the bundle/manifest format along the way.